portfolio/src/main/webapp/index.html

```diff
@@ -12,23 +12,88 @@
   </head>
   <body>
 
+    <div class="loader"></div>
+
     <div id="header"> 
         <div id="header-spacer"></div>
         <img class="headerels headerimages" src = "images/google.png"/>
         <p class="headerels">Seun Omonije STEP Portfolio</p>
         <input id="searchbar" class="headerels" type="text" placeholder="Search about me! (unimplemented)" />
     </div>
 
+    <div id ="contentspacer"></div>
 
     <div id="content">
-      <h1>Start</h1>
-      <p>This is my portfolio.</p>
+        <div id = "firstpart">
+            <div id="greeting-container">
+                <h1>About Me</h1>
+
+                <p> My name is Oluseun Omonije, but I go by Seun. I am a rising junior at Yale University, working towards my BS in Computer Science. 
```

Comment:
Multiline html should have its content indented on the next line like:
<p>
  Text
</p>